ci/travis/inside_docker.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+project_dir=$1
+pushd ${project_dir}
+
+cmake_option=$2
+
+GLOG_INSTALL_DIR="/aditof-deps/installed/glog"
+PROTOBUF_INSTALL_DIR="/aditof-deps/installed/protobuf"
+WEBSOCKETS_INSTALL_DIR="/aditof-deps/installed/websockets"
+
+mkdir -p build
+pushd build
+cmake .. ${cmake_option} -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="${GLOG_INSTALL_DIR};${PROTOBUF_INSTALL_DIR};${WEBSOCKETS_INSTALL_DIR}"
+make -j${NUM_JOBS}
+popd #build
+
+popd # ${PROJECT}
```

Comment:
PROJECT variable no longer exists

Reply:
The variable was renamed to project_dir, modified the comment.